Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -759,7 +769,7 @@ public SortDirection CurrentSortDirection
     /// <para>Sets the filter method to be used for filtering the column.</para>
     /// <para>If null, uses the <see cref="DataGrid{TItem}.FilterMethod" /> </para>
     /// </summary>
-    [Parameter] public DataGridFilterMethod? FilterMethod { get; set; }
+    [Parameter] public DataGridColumnFilterMethod? FilterMethod { get; set; }

```

Comment:
Isn't this a breaking change API since we're renaming the enum?

Reply:
Well yes. But the enum definitions are equivalent, so I was on the fence on this one. 
However I understand if we want to be rigid about that... 
We could just introduce an extra parameter and make that one obsolete?

Reply:
We can just add new fields to the `DataGridFilterMethod`, and remove `DataGridColumnFilterMethod`.

Reply:
I don't like that as much because it's also used as part of the general DataGrid filter, which I don't feel like would make as much sense to have those.

Reply:
Although that would definitely be easier & less complex, from a code standpoint, of course.

Reply:
I like the idea of having a separate DataGridColumnFilterMethod, just fear that we will introduce a _breaking_ API.

Reply:
So either
we add the the additional enums to `DataGridFilterMethod`, I guess there could still be some utility.... Even though it will not make sense for text columns... maybe a `DataGrid` that just is number based...? 

Or we add a new `Parameter` with `ColumnFilterMethod` name and make the other one obsolete and with compatibility.

I'm fine with both honestly, but I think I like the second option better, you can go ahead and choose haha.

Reply:
Go with DataGridColumnFilterMethod. YOLO!

Reply:
What do you mean? The current version?

Reply:
Yes. 1.4.